var/spack/repos/builtin/packages/qb3/package.py

```diff
@@ -18,3 +18,9 @@ class Qb3(CMakePackage):
 
     depends_on("cmake@3.5:", type="build")
     depends_on("libicd")
+
+    @property
+    def libs(self):
+        # Override because libs have different case than Spack package name
+        name = "libQB3*"
+        return find_libraries(name, root=self.prefix, shared=True, recursive=True)
```

Comment:
If we know where libraries are being installed, it would be good to set recursive to `False`

Reply:
Thanks for the suggestion.  I confess that I am new to messing with the libs method.  In this case (and openexr and sfcgal also), the libs are being installed to lib64 under prefix on my systems.  Presumably that would be just lib on a 32 bit system.    I would appreciate guidance re the best way to approach this (and will then update the 3 PRs):

1) Just assume everyone is using 64 bit and use lib64 (and might that break on some OSes which use lib even if they are 64 bit)?
2) Leave recursive=True?
3) set root=spec['hdf5'].libs.directories[0] (based  comments in #2617)

Since these packages should only have a single lib(64) directory, I am leaning towards 3 but figured I would ask for advice first.